### ISSUE TITLE
Images used in a repeat are only showing spinners

### ISSUE THREAD

**Issue by newtewt** [medic/cht-core#6450]
**Describe the bug**
When an image is used in a repeat block the image just shows a spinner. This can be reproduced using the form attached. Seems to happen on 3.8.1 and greated. 

**To Reproduce**
1. Upload the sample form and add an image.
2. Navigate to the form.
2. Enter value of 1 in the repeat section.
3. Click repeat
4. See the spinnners loading. There are 2 shown 1 for each translation. 

**Expected behavior**
The image should be shown. 

**Logs**
No logs in console.

**Screenshots**
![image](https://user-images.githubusercontent.com/10531066/83148797-61ebc980-a0c7-11ea-8a73-5781e694d52b.png)

**Environment**
- Instance: Localhost but @binokaryg  was using it somewhere for a partner I assume.
- Browser: Chrome
- Client platform: Linux
- App: webapp
- Version: 3.8.1 +

**Additional context**
I looked at the xml on 3.7 and master in the console. They matched as far as I could tell. Assume this might be fallout from #6321

**Comment by garethbowen** [comment]
@newtewt I can't see an attached form. Also, do you know if this worked before 3.8.1?

**Comment by newtewt** [comment]
Attached the form now. Forgot it.  I tried on 3.7 with it working and went to 3.8.1 with it not working.

```xml
<?xml version="1.0"?>
<h:html xmlns="http://www.w3.org/2002/xforms" xmlns:ev="http://www.w3.org/2001/xml-events" xmlns:h="http://www.w3.org/1999/xhtml" xmlns:jr="http://openrosa.org/javarosa" xmlns:orx="http://openrosa.org/xforms" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
  <h:head>
    <h:title>Repeat with image</h:title>
    <model>
      <itext>
        <translation lang="en">
          <text id="/repeat/image_out:label">
            <value>Image outside repeat group</value>
            <value form="image">jr://images/mark.png</value>
          </text>
          <text id="/repeat/num_repeat:label">
            <value>Repeat for how many times?</value>
          </text>
          <text id="/repeat/repeat/image_in:label">
            <value>Image inside repeat group</value>
            <value form="image">jr://images/mark.png</value>
          </text>
          <text id="/repeat/repeat:label">
            <value>REPEAT</value>
          </text>
        </translation>
        <translation lang="fr">
          <text id="/repeat/image_out:label">
            <value>Image outside repeat group</value>
            <value form="image">jr://images/mark.png</value>
          </text>
          <text id="/repeat/num_repeat:label">
            <value>Repeat for how many times?</value>
          </text>
          <text id="/repeat/repeat/image_in:label">
            <value>Image inside repeat group</value>
            <value form="image">jr://images/mark.png</value>
          </text>
          <text id="/repeat/repeat:label">
            <value>REPEAT</value>
          </text>
        </translation>
      </itext>
      <instance>
        <repeat delimiter="#" id="repeat" prefix="J1!repeat!" version="2020-05-28 18:22:41">
          <image_out/>
          <num_repeat/>
          <repeat_count/>
          <repeat jr:template="">
            <image_in/>
          </repeat>
          <meta tag="hidden">
            <instanceID/>
          </meta>
        </repeat>
      </instance>
      <instance id="contact-summary"/>
      <bind nodeset="/repeat/image_out" readonly="true()" type="string"/>
      <bind nodeset="/repeat/num_repeat" type="int"/>
      <bind calculate=" /repeat/num_repeat " nodeset="/repeat/repeat_count" readonly="true()" type="string"/>
      <bind nodeset="/repeat/repeat/image_in" readonly="true()" type="string"/>
      <bind calculate="concat('uuid:', uuid())" nodeset="/repeat/meta/instanceID" readonly="true()" type="string"/>
    </model>
  </h:head>
  <h:body>
    <input accuracyThreshold="1.5" ref="/repeat/image_out">
      <label ref="jr:itext('/repeat/image_out:label')"/>
    </input>
    <input ref="/repeat/num_repeat">
      <label ref="jr:itext('/repeat/num_repeat:label')"/>
    </input>
    <group ref="/repeat/repeat">
      <label ref="jr:itext('/repeat/repeat:label')"/>
      <repeat jr:count=" /repeat/repeat_count " nodeset="/repeat/repeat">
        <input ref="/repeat/repeat/image_in">
          <label ref="jr:itext('/repeat/repeat/image_in:label')"/>
        </input>
      </repeat>
    </group>
  </h:body>
</h:html>

```

**Comment by isaacholeman** [comment]
This is needed for the new cohort of covid surveillance CHWs that the Ministry of Health in Mali hopes to start training as early as June 5th. Specifically, they will be screening for malaria as well as covid symptoms, and they're expecting our typical visual guides for supporting malaria rapid diagnostic tests. 

@garethbowen @MaxDiz the level of priority here is pretty high, but probably not high enough to block the implementation because in the interim I think they could implement the form with text-only guidance if necessary. @binokaryg @Fatoufall correct me or add context if I'm incorrect here! 

The bug looks similar to #6321, but that one was fixed in 3.8.1. If this one doesn't appear until 3.8.1, it may be worth exploring whether they're somehow related.

**Comment by garethbowen** [comment]
There's another bug shown in the screenshot which is with multiple languages available you see multiple spinners until the image loads. This PR fixes that too.

**Comment by dianabarsan** [comment]
Ready for AT on `6450-replace-images-in-repeats`

**Comment by newtewt** [comment]
This is working good. Images are showing in the repeats fine. Also tried to reproduce the potentially related issue and did not see any spinners.  @garethbowen  feel free to close and merge.

**Comment by isaacholeman** [comment]
Fantastic that this was fixed so quickly, thank you! @kennsippell @garethbowen @abbyad would you confer with @binokaryg and decide what's the recommended implementation approach for MOH Mali? Go live date is this Friday June 5th.

**Comment by kennsippell** [comment]
I believe the current plan is to get this merged and into a `3.9-beta2` branch. Once that's ready, it's my opinion that the Covid-MoH-Mali development work and test pass should progress from that beta branch.

**Comment by garethbowen** [comment]
Merged and backported to `3.9.x`
